src/catalog.h

```diff
@@ -871,6 +871,12 @@ typedef struct FormData_continuous_agg
 	NameData user_view_name;
 	NameData partial_view_schema;
 	NameData partial_view_name;
+	/*
+	 * Don't access bucket_width directly to determine the width of the bucket.
+	 * Use corresponding procedures instead:
+	 * - ts_continuous_agg_bucket_width
+	 * - ts_continuous_agg_max_bucket_width
+	 */
```

Comment:
This comment will be hard to maintain if you list the functions. It is trivial for me to hit `M-?` to find all the locations where this field is used, so I would not worry about adding function names here.
```suggestion
	/*
	 * Don't access bucket_width directly to determine the width of the bucket.
	 */
```